docs/comparison.md

```diff
@@ -60,12 +57,12 @@
 <li>Streaming Metrics
 <li>Transaction Tracing
 <li>All Telegraf Input Plugins
+<li>Windows Collection
```

Comment:
I believe we should split it for specific sources, as we started to support part of them